client/src/components/front-page/CourseTableRow.js

```diff
@@ -17,11 +17,11 @@ const CourseTableRow = ({ course }) => {
     <TableRow
       key={id} 
       hover={true}
-      onClick={() => { navigate('/course-view/' + courseCode); }}
+      onClick={() => { navigate('/course-view/' + id); }}
```

Comment:
I added this comment line in my PR #173 to App.js:

<Route path='/course-view/:courseCode' element={<CourseView/>}/>
{ /* Path above will be replaced with '/course-view/:courseId' */ }

I think this change could be done in that branch once this branch is merged to main since I have done other actions in that branch as well (see comment above) so that merging this branch to that would be easier.